guides/getting_started_hassio.rst

```diff
@@ -30,55 +28,43 @@ After that, wait a bit until it is installed (this can take a while). Click "Sta
 You should now be greeted by a nice introduction wizard which will step you through
 creating your first configuration.
 
-.. figure:: images/hassio_start.png
+.. figure:: images/dashboard_empty.png
     :align: center
     :width: 95.0%
 
 The wizard will guide you through creating your first configuration and, depending on your browser, install it on your ESP device. You will need to name your configuration and enter your wireless network settings so that the ESP device can come online and can communicate with Home Assistant.
 
-.. raw:: html
```

Comment:
This seems helpful, why remove it? There's also a couple external users: https://www.google.com/search?q=getting_started_hassio.html%23webserial

Reply:
Because if you are following the getting started and using the dashboard, then basically all the same note text is now contained inside there.